### DISCUSSION TITLE
Ремешок для оружия

### DISCUSSION THREAD

**Discussion by KillerBunny031** [octothorp-team/public-dobrograd#757] [Предложения]
Было бы интересно увидеть данную фичу. Сейчас, чтобы ходить с каким-то тяжелым оружием тебе надо или держать его в инвентаре, что приводит к запрету на бег или в руках, но тогда это уже боевая стойка, когда ты готов стрелять. Так вот. Можно было бы ввести в магазин саму штуку "Ремешок для оружия" и держать оружие за спиной. Интересно услышать мнение остальных людей по поводу этого. (актуально для всяких Альф, чтобы не отыгрывать каждую минуту /it винтовка за спиной)
___
![image](https://user-images.githubusercontent.com/87093007/143457088-72ff3d6a-9714-40b0-82ef-b1decfdf238f.png)
![image](https://user-images.githubusercontent.com/87093007/143457112-35c7b8b9-fcea-4999-8400-576d01e79ec3.png)

**Comment by travka4000** [comment]
тут честно более я за
